examples/loftq_finetuning/quantize_save_load.py

```diff
@@ -180,6 +180,22 @@ def quantize_and_save():
 
     print_model(base_model, "base_model")
 
+    # convert safetensor to bin
+    tensors = {}
+    with safe_open(os.path.join(lora_model_dir, "adapter_model.safetensors"), framework="pt") as f:
+        for k in f.keys():
+            tensors[k] = f.get_tensor(k)
+    torch.save(tensors, os.path.join(lora_model_dir, "adapter_model.bin"))
+
+    # change adapter_config.json
```

Comment:
Instead of loading and overwriting the file, would it make more sense to set the attributes correctly before even saving the adapter?